megamock/megamocks.py

```diff
@@ -38,6 +46,14 @@ def __call__(self, target_callable: Callable) -> None:
 UseRealLogic = _UseRealLogic()
 
 
+def set_return_value(func: Callable, value: Any) -> None:
```

Comment:
Attempting to set `return_value` as an attribute when its been cast to the underlying object isn't going to play nice with mypy unless you cast first. This is burdensome, so this function lets you set the return value.

Reply:
One consideration is that you have:

```
UseRealLogic(something...)
```

and

```
set_return_value(something...)
```

which isn't consistent. It might be worth making these both the same types, likely functions